src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -330,13 +344,25 @@ def read_forward_model(
             try:
                 job = copy.deepcopy(installed_jobs[job_name])
             except KeyError as err:
-                raise ConfigValidationError(
-                    errors=(
-                        f"Could not find job {job_name!r} in list of installed jobs: "
-                        f"{list(installed_jobs.keys())!r}"
-                    ),
-                    config_file=config_file,
-                ) from err
+                errors.append(
+                    ConfigValidationError(
+                        info=ErrorInfo(
+                            message="",
+                            data={},
```

Comment:
Rather than having a dummy ErrorInfo,  I think we should leave it as None for now.